app/models/course/question_assessment.rb

```diff
@@ -7,6 +7,18 @@ class Course::QuestionAssessment < ApplicationRecord
 
   default_scope { order(weight: :asc) }
 
+  # Prefixes a question number in front of the title
+  #
+  # @return [string]
+  def display_title
```

Comment:
Actually it might be possible that `QuestionAssessment` have a title, separate from `Question`. But handling the behavior can be a little tricky >_<